cmake/hip.cmake

```diff
@@ -0,0 +1,46 @@
+if(NOT WITH_AMD_GPU)
+    return()
+endif()
+
+include_directories("/opt/rocm/include")
+include_directories("/opt/rocm/hipblas/include")
+include_directories("/opt/rocm/hiprand/include")
+include_directories("/opt/rocm/rocrand/include")
+include_directories("/opt/rocm/rccl/include")
+include_directories("/opt/rocm/thrust")
```

Comment:
Could `/opt/rocm/include` etc be a relative path like `${CUDA_INCLUDE_DIRS}`?

Reply:
Until now no such thing is defined in HIP/ROCm.

Reply:
No, we don't have such define. NV defines `/usr/local/cuda` as the symbolic link, that's why they need those env vars so users can define those to point to the absolute CUDA paths.